esphome/components/mopeka_pro_check/sensor.py

```diff
@@ -55,6 +54,7 @@ def small_distance(value):
     "EUROPE_11KG": (38, 366),
     "EUROPE_14KG": (38, 467),
 }
+SIGNAL_QUALITIES = {"ZERO": 0, "LOW": 1, "MEDIUM": 2, "HIGH": 3}
```

Comment:
You will need to also move this down lower in the file (below at least `mopeka_pro_check_ns`)
```suggestion
SensorReadQuality = mopeka_pro_check_ns.enum("SensorReadQuality")

SIGNAL_QUALITIES = {
    "ZERO": SensorReadQuality.QUALITY_ZERO,
    "LOW": SensorReadQuality.QUALITY_LOW,
    "MEDIUM": SensorReadQuality.QUALITY_MEDIUM,
    "HIGH": SensorReadQuality.QUALITY_HIGH,
}
```